test/e2e/storage/nfs_persistent_volume-disruptive.go

```
@@ -280,5 +280,7 @@ func tearDownTestCase(c clientset.Interface, f *framework.Framework, ns string,
 	// Ignore deletion errors.  Failing on them will interrupt test cleanup.
 	framework.DeletePodWithWait(f, c, client)
 	framework.DeletePersistentVolumeClaim(c, pvc.Name, ns)
-	framework.DeletePersistentVolume(c, pv.Name)
+	if pv != nil {
+		framework.DeletePersistentVolume(c, pv.Name)
```

ditto, err is ignored